src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -1397,6 +1403,50 @@ methodHandle SharedRuntime::find_callee_method(TRAPS) {
   return callee_method;
 }
 
+JRT_BLOCK_ENTRY(void, SharedRuntime::end_training_check_c1(JavaThread* current))
+{
+  JRT_BLOCK
+    SharedRuntime::end_training_check(CHECK);
+  JRT_BLOCK_END
+}
+JRT_END
+
+JRT_BLOCK_ENTRY(void, SharedRuntime::end_training_check_c2(JavaThread* current))
+{
```

Comment:
'{' at 1407
'}' at 1411 are redundant as JRT_BLOCK_ENTRY, JRT_BLOCK def includes '{' and JRT_BLOCK_END, JRT_END replaces '}'

Reply:
right but the existing code patterns (either for readability or because they didn't realize there are extra '{' and '}') is the way its written, could be in certain configurations that there's code added that needs to be scoped so it doesnt conflict with the method code